src/stats/overall.ts

```diff
@@ -174,7 +187,8 @@ function generateInputs(game: SlippiGame) {
 
     // Do the same for c-stick
     const prevCstickRegion = getJoystickRegion(prevPlayerFrame.cstickX, prevPlayerFrame.cstickY);
-    const currentCstickRegion = getJoystickRegion(playerFrame.cstickX, playerFrame.cstickY);
+    // FIXME: stop using any. it is not clear what type thins is. cstickX does not exist on the type PreFrameUpdateType
+    const currentCstickRegion = getJoystickRegion((playerFrame as any).cstickX, (playerFrame as any).cstickY);
```

Comment:
This actually looks like a bug lol. `cstickX` and `cstickY` do not exist on the pre-frame type, but `cStickX` and `cStickY` do.

Reply:
And that right there would be one argument for using typescript lol

Reply:
Yeah I didn't want to change it to `cStickX` and `cStickY` since I wasn't sure if it would have any unintended consequences. Also since it's not really a Typescript issue the fix should probably not be a part of this PR.